public/index.html

```diff
@@ -90,7 +90,7 @@
             <li>jQuery</li>
           </ul>
         </section>
-            <div id="timeTaken">Time taken total</div>
+            <div id="timeTaken">Time taken total for Project:</div>
```

Comment:
**TODO FOR LOST POINTS** 

Remember what I said about camel-case vs. kabob-case in HTML attributes. 

Also, I can't make a comment since there is no change, but please take out scripts/projectInfo.js now that you've determined you definitely don't need it. A point also got docked for that.